src/main/java/stelnet/commodity/view/CommodityViewFactory.java

```diff
@@ -1,23 +1,18 @@
 package stelnet.commodity.view;
 
-import java.util.Arrays;
-import java.util.List;
-
 import com.fs.starfarer.api.campaign.econ.MarketAPI;
-
 import org.lwjgl.input.Keyboard;
-
 import stelnet.commodity.CommodityBoard.CommodityTab;
-import stelnet.commodity.data.BuyMarketFactory;
-import stelnet.commodity.data.BuyTableContent;
 import stelnet.commodity.data.MarketFactory;
-import stelnet.commodity.data.SellMarketFactory;
-import stelnet.commodity.data.SellTableContent;
-import stelnet.ui.Renderable;
-import stelnet.ui.Row;
-import stelnet.ui.Stack;
-import stelnet.ui.Table;
-import stelnet.ui.TableContent;
+import stelnet.commodity.data.content.buy.BuyMarketFactory;
+import stelnet.commodity.data.content.buy.BuyTableContent;
+import stelnet.commodity.data.content.profit.ProfitTableContent;
+import stelnet.commodity.data.content.sell.SellMarketFactory;
+import stelnet.commodity.data.content.sell.SellTableContent;
+import stelnet.ui.*;
+
+import java.util.Arrays;
```

Comment:
Your formatter still sorts this differently. Also looks like it's removing new lines before groups

Reply:
It's still the old PR, I have not yet created updated one.

Reply:
I see. Keeping it open was fine, you could always rebase and force push. I was going to squash merge it when done so don't worry about git history too much.